modules/helm/crds.mk

```diff
@@ -37,29 +37,31 @@ ifeq ($(HOST_OS),darwin)
 	sed_inplace := sed -i ''
 endif
 
+crds_dir ?= deploy/crds
+crds_dir_readme := $(dir $(lastword $(MAKEFILE_LIST)))/crds_dir.README.md
+
 .PHONY: generate-crds
 ## Generate CRD manifests.
 ## @category [shared] Generate/ Verify
 generate-crds: | $(NEEDS_CONTROLLER-GEN) $(NEEDS_YQ)
-	$(eval crds_gen_temp := $(bin_dir)/scratch/crds)
 	$(eval directories := $(shell ls -d */ | grep -v -e 'make' $(shell git check-ignore -- * | sed 's/^/-e /')))
 
-	rm -rf $(crds_gen_temp)
-	mkdir -p $(crds_gen_temp)
+	mkdir -p $(crds_dir)
+	cat $(crds_dir_readme) > $(crds_dir)/README.md
```

Comment:
nitpick: The makefile would be a little more efficient if `$(crds_dir)/README.md` was its own target, since its contents depend only on one file - we don't need to update the readme every time we run `generate-crds`, we only need to update the README if `crds_dir.README.md` has changed.

Pseudo code:

```make
$(crds_dir):
<tab> mkdir -p $@

$(crds_dir)/README.md: path/to/crds_dir.README.md | $(crds_dir)
<tab> cp $< $@

generate-crds: $(crds_dir)/README.md | $(NEEDS_CONTROLLER-GEN) $(NEEDS_YQ) $(crds_dir)
...
```

Reply:
Great suggestion, Ashley! I have to understand a bit more about how the paths are set in makefile-modules, but this should be doable! Thanks!

Reply:
I think I've got this working now. PTAL!